lib/chkhash.h

```diff
@@ -0,0 +1,9 @@
+#ifndef _CHKHASH_H
+#define _CHKHASH_H
+
+#include <stdbool.h>
+#include <stddef.h>
+
+extern bool is_valid_hash (const char *hash);
```

Comment:
Please remove the space before `(`.

Reply:
And remove `extern`.  We don't write that in new functions.

Reply:
Also, please use two blank lines between blocks (before includes, before prototypes, after prototypes).